### PR TITLE
inventory model added

### PR DESCRIPTION
**Pull request by JohnWCCI**
This should be an inventory model

### CODE REVIEW

**Comment by travisruopp** [comment]
Comment

**Comment by Jfrnzn** [comment]
no

**Comment by JohnWCCI** [comment]
For get it rejected

**Comment by safilotcha** [comment]
cool

File: src/InventoryModel.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+public class InventoryModel {
+    private String Name;
+    private double price;
+    public InventoryModel(String name, double price) {
+        Name = name;
+        this.price = price;
+    }
+    public String getName() {
+        return Name;
```
Comment (JohnWCCI) [inline src/InventoryModel.java:9]: dummy

File: src/InventoryModel.java
Comment (Jfrnzn) [inline src/InventoryModel.java:1]: comment

**Review by JohnWCCI — COMMENTED** [review]
good

File: src/InventoryModel.java
Comment (travisruopp) [reply src/InventoryModel.java:1]: O_O

**Pull request merged**